jwql/website/apps/jwql/bokeh_dashboard.py

```diff
@@ -0,0 +1,232 @@
+"""Bokeh based dashboard to monitor the status of the JWQL Application.
+The dashboard tracks a variety of metrics including number of total
+files per day, number of files per instrument, filesystem storage space,
+etc.
+
+The dashboard also includes a timestamp parameter. This allows users to
+narrow metrics displayed by the dashboard to within a specific date
+range.
+
+Authors
+-------
+
+    - Mees B. Fix
+
+Use
+---
+
+    The dashboard can be called from a python environment via the
+    following import statements:
+    ::
+
+      from bokeh_dashboard impoer GeneralDashboard
+      from monitor_template import secondary_function
+
+Dependencies
+------------
+
+    The user must have a configuration file named ``config.json``
+    placed in the ``utils`` directory.
+
+"""
+
+from datetime import date
+from datetime import datetime as dt
+from math import pi
+from operator import itemgetter
+from random import randint
+
+from bokeh.embed import components
+from bokeh.io import output_file, show
+from bokeh.layouts import grid, column, row, layout, widgetbox
+from bokeh.models import ColumnDataSource, DatetimeTickFormatter, OpenURL, TapTool
+from bokeh.models.widgets import DataTable, DateFormatter, TableColumn, Panel, Tabs
+from bokeh.plotting import figure
+from bokeh.transform import cumsum
+import numpy as np
+import pandas as pd
+
+from jwql.database.database_interface import FilesystemInstrument, load_connection, Monitor
+from jwql.utils.constants import FILTERS_PER_INSTRUMENT
+from jwql.utils.utils import get_base_url, get_config
+from jwql.website.apps.jwql.data_containers import build_table
+
+
+class GeneralDashboard:
+
+    def __init__(self, delta_t=None):
+        self.name = 'jwqldb_general_dashboard'
+        self.delta_t = delta_t
+
+        now = dt.now()
+        self.date = pd.Timestamp('{}-{}-{}'.format(now.year, now.month, now.day))
+
+
+    def dashboard_filetype_bar_chart(self):
+        """Build bar chart of files based off of type"""
```

Comment:
It looks like most of the methods in this module are missing `Parameters` and/or `Returns` sections in their docstrings.  Could you add those?